diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter2/KdocMethodsTest.kt

```diff
@@ -425,4 +425,28 @@ class KdocMethodsTest : LintTestBase(::KdocMethods) {
             DiktatError(2, 1, ruleId, "${MISSING_KDOC_ON_FUNCTION.warnText()} writeToConsoleEx", true),
         )
     }
+
+    @Test
+    @Tag(WarningNames.MISSING_KDOC_ON_FUNCTION)
+    fun `KDoc shouldn't trigger on local functions`() {
+        lintMethod(
+            """
+                |fun printHelloAndBye() {
```

Comment:
@diphtongue you don't like to write tests, I see 😄 
But pls write some more cases: for example with several nested methods.

And added both scenarios: when Kdoc is presented and when is not

Reply:
Done